homeassistant/components/reolink/update.py

```diff
@@ -196,6 +197,8 @@ async def async_install(
         try:
             await self._host.api.update_firmware(self._channel)
         except ReolinkError as err:
+            if err.translation_key:
```

Comment:
How can we ensure all possible translation keys are present in the `strings.json`?

 One solution could be that we change the type of `translation_key` to a `StrEnum` and add a test to check if all members are in strings.json

Reply:
Very good point.

I like the idea of a StrEnum, although I think a set of strings (the keys) would maybe be even easier.

Do you know if there is a standard way to load the strings.json into a test?

Reply:
Or maybe when there is an error I could dynamically load the string.json file

Reply:
I made a automated check if the translation key is present or not.
In this way we dont need to worry about a translation key in the upstream library which is not present in strings.json